webrtc-stats.html

```diff
@@ -1218,6 +1218,16 @@
              double                        roundTripTime;
              double                        availableOutgoingBitrate;
              double                        availableIncomingBitrate;
+             unsigned long long            requestsReceived;
+             unsigned long long            requestsSent;
+             unsigned long long            responsesReceived;
+             unsigned long long            responsesSent;
+             unsigned long long            retransmissionsSent;
```

Comment:
If the implementation doesn't retransmit, then retransmissionsSent will always be zero.  For such an implementaiton, the pre and post-consent transport behavior is the same.